README.md

```diff
@@ -603,6 +603,18 @@ Uhm, the best. It's awesome. Stop what you're doing and go watch it. Now. - [@Ri
 
 > In a war-torn world of elemental magic, a young boy reawakens to undertake a dangerous mystic quest to fulfill his destiny as the Avatar.
 
+#### [Death Note](https://www.imdb.com/title/tt0877057/) [9.0]
+
+One of the best manga series i have ever seen, one of my favourite (2015 TV series) Death Note (デスノート) is a Japanese television drama series based on the manga series of the same name by Tsugumi Ohba and Takeshi Obata Watch now it's too good - [@RichardLitt](https://github.com/RichardLitt)
```

Comment:
I didn't say this. Can you add your name, instead?

Reply:
Surely i can, but don't you think this can be helpful as death note is a TV series also but then also if you want to change it to my name then please do tell me, not an issue

Reply:
I'm more concerned that you're quoting me, as opposed to quoting yourself. :P

Reply:
Oh ya forgot about that, i will fix it

Reply:
Done the work